model_definition.md

```diff
@@ -0,0 +1,375 @@
+## Model overview: wastewater-informed, site-level infection dynamics
+
+This document details the generative model used by `wwinference` to infer global and local infection dynamics from count data (e.g. cases or hospital admissions) and wastewater concentration data, and use that to produce nowcasts and forecasts.
+The `wwinference` model assumes that wastewater concentration data is available for one or more "local" sites that represent a subset of the total population that produce the "global" epidemiological indicators (cases or admissions).
+In the future, we plan to provide the user with functionality for other types of data structures, e.g. multiple data streams of hospital admissions in addition to multiple wastewater concentration data streams, but for now this structure is the only option provided to the user.
+The model is structured into four generative components: [infections](#infection-component), [hierarchical subpopulation-level infection dynamics](#hierarchical-subpopulation-infection-dynamics-component), [hospital admissions](#hospital-admissions-component), and [viral genome concentrations in wastewater](#viral-genome-concentration-in-wastewater-component).
+The model imposes a hierarchical structure where the infection dynamics in the subpopulations represented by the wastewater concentration data are assumed to be localized outbreaks similar to one another and centered around the "global" infection dynamics that give rise to the hospital admissions. Note, we will describe the model in terms of the generation of hospital admissions, but the user can choose to replace this with any "count" dataset with a delay distribution from infection to the generation of that count data, e.g. cases would also work well here.
+
+### Model components
+
+Our models are constructed from a set of generative components. These are:
+
+- [**Infection component:**](#infection-component) A renewal model for the infection dynamics, which generates estimates of incident latent infections per capita.
+- [**Hierarchical subpopulation-level infection dynamics:**](#hierarchical-subpopulation-infection-dynamics-component) A model for the relation between the subpopulation infection dynamics and the "global" infection dynamics
+- [**Hospital admissions component:**](#hospital-admissions-component) A model for the expected number of hospital admissions given incident latent infections per capita.
+- [**Viral genome concentration in wastewater:**](#viral-genome-concentration-in-wastewater-component) A model for the expected genome concentration given incident infections per capita.
+
+The general flow of the model can be visualized as follows:
+
+```mermaid
+flowchart LR;
+    GlobalR --> GlobalI;
+    GlobalR --> LocalR;
+    LocalR --> GlobalR;
+    LocalR --> LocalIX;
+    LocalR --> LocalIY;
+    LocalIX --> LocalWWX;
+    LocalIY --> LocalWWY;
+    GlobalI --> Globalhosp;
+```
```

Comment:
Don't you want to show `Global I` as the sum of the `Local I`, not as a function of `Global R`? What we have now looks like a model without bottom-up infection aggregation.

Also note that you can use $\TeX$-like math notation, so you can have e.g. $\mathcal{R}$ if you want.

Reply:
Ooh ok let me try to use the Latex formatting. I did think about that, I need to figure out how to put it in the diagram (maybe a box around the two Is with an arrow pointing to the Global I), and then remove the connection from global R to global I?

Reply:
I tried using `Global$R$` and just got the $. I think @gvegayon said that github might not support this yet even though mermaid does?

Reply:
That's right, last time I checked, GitHub doesn't support all features. I don't think it supports math yet, but it does markdown!:

```mermaid
flowchart LR;
  X["`This should be __bold__ and _cursive_`"]
  Y["This should print the delta symbol $$\delta$$"]
```

## Session info version

```mermaid
info
```